docker/Dockerfile

```diff
@@ -10,9 +10,11 @@ WORKDIR /tmp
 RUN unzip -q /tmp/rdf4j.zip
 
 # Final workbench
-FROM tomcat:8.5-jre11-temurin AS wb
+FROM tomcat:8.5-jre11-temurin
 MAINTAINER Bart Hanssens (bart.hanssens@bosa.fgov.be)
 
+RUN apt-get update && apt-get upgrade -y
```

Comment:
This forces an update of all linux packages.

Reply:
Perhaps add a `&& apt-get clean` to clean the download cache (should save some space on the image)